create-users-projects.yml

```diff
@@ -2,9 +2,9 @@
 - name: Create users and projects
   hosts: localhost
   roles:
-    - role: users-projects
-      tags: ['create-hosts']
-      os_cloud: opentech-sjc
+    - role: cloud-bootstrap
+      tags: ['create-users-projects']
```

Comment:
If we're going to make this more of a bootstrap role, we should remove the users-projects tag off of the role itself, and instead rely on the tags inside the role.

Should probably re-name the play file too.